app/views/effective/alerts/_alert.html.haml

```diff
@@ -0,0 +1,4 @@
+%div{class: "alert alert-warning", role: 'alert'}
```

Comment:
Aah yea, alert-warning. I wonder if this will need to get tweaked. For now we're keeping it simple so ignore :)